package.json

```diff
@@ -37,20 +37,20 @@
     "typescript": "^3.9.3"
   },
   "peerDependencies": {
-    "eslint": "^6.4.0",
-    "typescript": "^3.3.3333"
+    "eslint": "^7.0.0",
+    "typescript": ">=3.3.1 <3.10.0"
```

Comment:
This versions referenced to [typescript-eslint#supported-typescript-version](https://github.com/typescript-eslint/typescript-eslint#supported-typescript-version)

Reply:
@toshi-toma Could you suggest this again because of stale?